src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidateNeverAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// Indicates that a property should be excluded from validation. When applied to a property, the validation
+    /// system excludes that property. When applied to a type, the validation system excludes all properties within
+    /// that type.
```

Comment:
This still applies w/ `IShouldValidate` and the new `[ValidateNever]`. But, I'm not sure I agree this is a bad idea. For one thing, the choices exactly match how `[BindNever]` et cetera can be used. For another, `ValidateChildren = false` and its `SuppressChildValidationMetadataProvider` wrapper _configure_ something that we use metadata for in most analogous situations.

As I said [here](https://github.com/aspnet/Mvc/pull/5676#pullrequestreview-16269449), my recommendation would be to use `[ValidateNever]` on types you own and `SuppressChildValidationMetadataProvider` on types you don't.